README.rst

```diff
@@ -26,11 +26,19 @@
 SKOPS
 =====
 
-This library helps you share your scikit-learn based models and put them in
-production.
+``skops`` is a Python library helping you share your `scikit-learn
+<https://scikit-learn.org/stable/>`__ based models and put them in production.
+At the moment, it includes the following features:
+
+- ``skops.hub_utils``: tools to create a model repository to be stored on
+  `Hugging Face Hub <https://hf.co/models>`__, mainly through
```

Comment:
> mainly through ``skops.hub_utils.init`` and ``skops.hub_utils.push``.

I think this doesn't add much for the reader. I think the added value (apart from storing and sharing) rather comes from the inference widget. WDYT?